server/src/lib/model/db.ts

```diff
@@ -130,4 +130,22 @@ export default class DB {
   endConnection(): void {
     this.mysql.endConnection();
   }
+
+  async findClipsWithFewVotes(limit: number): Promise<DBClipWithVoters[]> {
```

Comment:
Opinion: I think this query should like in the clips-table object (and this db function should query to a similar function there). The weird part is that it's a join, but it is a query based on clips (which just pulls in data from votes), and I think we want these things in the table objects.

Reply:
I thought db would be a good place to have those kind of fns for now because 
- given that a migration library is still on the table, the future of the `*-table.js` files seems uncertain
- fewer middlemen forwarding method calls